zerver/tests/test_realm.py

```diff
@@ -141,14 +140,6 @@ def test_do_deactivate_realm_clears_user_realm_cache(self):
         user = self.example_user('hamlet')
         self.assertTrue(user.realm.deactivated)
 
-    def test_do_deactivate_realm_clears_scheduled_jobs(self):
-        # type: () -> None
-        user = self.example_user('hamlet')
-        send_future_email('template_prefix', user.email, delay=datetime.timedelta(hours=1))
-        self.assertEqual(ScheduledJob.objects.count(), 1)
-        do_deactivate_realm(user.realm)
-        self.assertEqual(ScheduledJob.objects.count(), 0)
-
```

Comment:
why are you deleting this test?  It was added recently, so my guess is you resolved a merge conflict incorrectly?  The changes at the top of the file are probably also not intentional.